src/Ninja/G1CP/Content/Fixes/Gamesave/fix0057_CastleWallTexture.d

```diff
@@ -1,38 +1,39 @@
 /*
  * #57 Misplaced texture near Gravo
  */
-func int G1CP_0057_CastleWallTexture() {
-    var int vobPtr; vobPtr = G1CP_FindVobByPosF(-5893.91455, -664.270386, 78.9339142, G1CP_0057_CastleWallTextureCheck);
-    if (vobPtr) {
-        G1CP_MoveVobToPosF(vobPtr, -5902.01807, -664.270386, 82.8507462);
-        return TRUE;
-    };
-    return FALSE;
-};
+
+/*
+ * Make the positions available to the functions below
+ */
+const float G1CP_0057_CastleWallTexture_OriginalPos[3] = {-5893.91455, -664.270386, 78.9339142};
+const float G1CP_0057_CastleWallTexture_CorrectedPos[3] = {-5902.01807, -664.270386, 82.8507462};
 
 /*
  * Callback function to check the found VOBs: Check if the VOB's visual is a decal
  */
 func int G1CP_0057_CastleWallTextureCheck(var int vobPtr) {
     var zCVob vob; vob = _^(vobPtr);
-    const int zCDecal_classDef = 8861528; //0x873758
+    const int zCDecal_classDef = 8861528; // 0x873758
```

Comment:
Fun fact: Although odd-looking at first, it kind of established itself to have no leading space for comments that describe memory addresses. This is not really a convention, but I like to stick to how Ikarus started it 😄